be/src/exec/broker_scanner.cpp

```diff
@@ -158,53 +143,11 @@ Status BrokerScanner::open_file_reader() {
             _skip_lines = 2;
         }
     }
-    switch (range.file_type) {
```

Comment:
I think it is better to use stream type not file type. 
FileType maybe parquet, orc, csv
StreamType could be S3Stream, HDFSStream, LocalStream

Reply:
Since the original code set the type in thrift, this name must be reserved for compatibility